style/desktop-style.css

```diff
@@ -0,0 +1,133 @@
+@media (min-width: 768px) {
+  body {
+    max-width: 144rem;
+    min-height: auto;
+    font-size: 1rem;
+  }
+
+  .flex-container {
+    background-image: url(../images/Header.png);
+    background-size: cover;
+    padding: 160px 260px 90px 130px;
+  }
+
+  .my-button {
+    display: none;
+  }
+
+  .header ul {
+    list-style: none;
+    display: flex;
+    justify-content: end;
+    gap: 1rem;
+    width: 100%;
+  }
+
+  .name {
+    display: flex;
+    margin: 0;
+  }
+
+  .card-container {
+    min-width: 100%;
+  }
+
+  .card {
+    display: flex;
+    min-width: 30rem;
+  }
+
+  .cardpic {
+    width: 120%;
+    height: 96%;
+  }
+
+  .card1-description-p {
+    display: none;
+  }
+
+  .card1-description-alt-p {
+    display: flex;
+    font-size: 2.5rem;
+  }
+
+  .card-title {
+    display: none;
+  }
+
+  .card-title-two {
+    display: flex;
+    font-style: normal;
+    font-weight: 500;
+    font-size: 2rem;
+    line-height: 5rem;
+    color: #091e42;
+    padding: top 0;
+    margin: 0;
+  }
```

Comment:
- [ ] Kindly make sure you improve your **projects cards** styling to make them stick to the Figma design. I noticed that from Figma template elements in each block in a card used half of the `width` i.e. **Image block width 50%** and **text block 50%**. I suggest making the same on your project
- [ ] Also, make sure to use the correct `spacing` around the `cards` (**spacing at the right and left of your cards**) as from the Figma design. And your cards will stick to the template design.
#### Figma design
![card_fig](https://user-images.githubusercontent.com/83042742/173109725-ef8dffed-a71b-4d43-ae34-275ebe7ff646.png)

#### Your design
![card_st](https://user-images.githubusercontent.com/83042742/173109638-8553c18d-ce08-42af-8833-d58ec36e13e6.png)